### PR TITLE
Update versions and add dsl support

### PR DESCRIPTION
**Pull request by mam10eks**
With this pull request I update the version of jmh-visualizer to 0.9.
Also I add a dsl-method (in order that one could use this plugin with the jenkins-job-dsl).

An working dsl example would be:

```
jmhReport {
    resultPath('target/reports/jmh/result.json')
}
```

### CODE REVIEW

File: src/main/webapp/provided.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+// provided.js can be overriden in order to have a predefined-report generated.
```
Comment (jzillmann) [inline src/main/webapp/provided.js:1]: Why do you added the provided.js ?

File: src/main/webapp/settings.js [hunk 2]
```diff
@@ -1,3 +1,3 @@
-var settings = { // eslint-disable-line no-unused-vars
-    showHeader: false
+var defaultSettings = { // eslint-disable-line no-unused-vars
+    topBar: 'default' //possible values ['default', 'off', 'my custom headline']
```
Comment (jzillmann) [inline src/main/webapp/settings.js:2]: Will look into this, but i guess it should be 'off' by default (matches showHeader=false then).

**Review by jzillmann — COMMENTED** [review]
Hey @mam10eks ,

thanks for the contribution!
I'm not a Jenkins pro.. Is there anything you would add to the Readme under the "Setup in Jenkins" section for the job-dsl thing ?

File: src/main/webapp/provided.js (hunk 1 above)
Comment (mam10eks) [reply src/main/webapp/provided.js:1]: There is no reason.
I simply checked out [JMH-Vizualizer](https://github.com/jzillmann/jmh-visualizer) and build it.
After that I moved all build-artifact's into the associated resources directory.

After that the plugin-ui has some new features which are working and looking nice.
Hence I thought everything is fine.

If some of the resources are unnecessary you could remove it or adopt it (like the settings.js).

**Comment by mam10eks** [comment]
Hey @jzillmann ,
thanks for your feedback.

I will add some points to the "Setup In Jenkins"-Section soon.

**Comment by jzillmann** [comment]
Dankeschön!

I separated the jmh-visualizer update from the Job DSL stuff and brought everything to master and released it as 0.9!

**Pull request closed without merging**